src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -168,7 +167,7 @@ const BaseComment = (props: Props) => {
     const isResolved = status === COMMENT_STATUS_RESOLVED;
 
     return (
-        <ActivityCard className="bcs-Comment">
+        <div className="bcs-Comment">
```

Comment:
Can we add a `data-testid`?

Reply:
We should add data-testid attributes when they are needed. In this case, we can check the actual content of the comment. If we can, we should always test what the user would see with rtl.